db-integrity-tests/src/gremlin_configuration.py

```diff
@@ -0,0 +1,15 @@
+"""Configuration for the connection to the Gremlin service."""
+import os
+
+
+class GremlinConfiguration:
+    """Class representing configuration for the connection to the Gremlin service."""
+
+    def __init__(self):
+        """Construct the configuration structure from the environment variables."""
+        self.url = os.environ.get('F8A_GREMLIN_URL')
+
+        # check if the required environment variable has been set
+
+        assert self.url is not None, \
```

Comment:
rather just `assert self.url` (since `F8A_GREMLIN_URL` is by default set to empty string in `runtest.sh`)